packages/hoppscotch-common/src/components/http/Authorization.vue

```diff
@@ -233,15 +232,6 @@ const emit = defineEmits<{
 
 const auth = useVModel(props, "modelValue", emit)
 
-onMounted(() => {
-  if (props.isRootCollection && auth.value.authType === "inherit") {
-    auth.value = {
-      authType: "none",
-      authActive: true,
-    }
-  }
-})
-
```

Comment:
I think we can keep this since a root collection can't inherit auth so we should set the auth to "None" in this case.

Reply:
also for a newly created collection the auth active is disabled by default can you change it to active

Reply:
We always keep the authType to `inherit` even if the request isn't in any collection (it will work like none). That we have discussed already with @thetronjohnson